R/RunModule.R

```diff
@@ -28,7 +28,11 @@ runModule <- function(analysisSpecifications, keyringSettings, moduleIndex, exec
   version <- moduleSpecification$version
   remoteRepo <- moduleSpecification$remoteRepo
   remoteUsername <- moduleSpecification$remoteUsername
-  moduleFolder <- ensureModuleInstantiated(module, version, remoteRepo, remoteUsername)
+  moduleInstallation <- verifyModuleInstallation(module, version)
```

Comment:
This call is repeated a few times so may be worth replacing

Reply:
So the calls are repeated for the different operations (running a module, creating results schema, uploading results). We can make a separate issue to define an approach that consolidates the common elements of these functions to avoid this repetition in the code base. Let me know what you think.